src/helpers/helperUtils.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable valid-jsdoc */
```

Comment:
Why disabled this?

Reply:
It was configured in eslint, so without that eslint will flag as error